servers/graphql-kotlin-ktor-server/src/main/kotlin/com/expediagroup/graphql/server/ktor/GraphQLRoutes.kt

```diff
@@ -56,14 +57,24 @@ fun Route.graphQLGetRoute(endpoint: String = "graphql", streamingResponse: Boole
  * @param endpoint GraphQL server POST endpoint, defaults to 'graphql'
  * @param streamingResponse Enable streaming response body without keeping it fully in memory. If set to true (default) it will set `Transfer-Encoding: chunked` header on the responses.
  * @param jacksonConfiguration Jackson Object Mapper customizations
+ * @param acceptContentType Only route requests with accept headers that match this parameter
+ * @param responseContentType Respond by this parameter
  */
-fun Route.graphQLPostRoute(endpoint: String = "graphql", streamingResponse: Boolean = true, jacksonConfiguration: ObjectMapper.() -> Unit = {}): Route {
+fun Route.graphQLPostRoute(
+    endpoint: String = "graphql",
+    streamingResponse: Boolean = true,
+    jacksonConfiguration: ObjectMapper.() -> Unit = {},
+    acceptContentType: ContentType = ContentType("application", "graphql-response+json"),
```

Comment:
we probably should wait for ktor 2.3.0 which includes https://github.com/ktorio/ktor/commit/45b4ccd895d75cc746b8c4b907d5b7730408b0b2

how do we also provide a fallback to just use `application/json` if there is no match?

Reply:
OK. I removed support in ktor server from this PR.
I'll make another PR when ktor 2.3.0 is released. 👍